Tutorials/securing.xml

```diff
@@ -1,6 +1,6 @@
 <PropertyList>
 
-    <name>Securing aircraft</name>
+    <name>Securing Aircraft</name>
 
     <description>
 After the landing and before you go having a beer you need to secure the aircraft.
```

Comment:
I am not a big fan of this pun either. Would you mind removing it? Since this PR was created in your clone I can't do it myself.

Reply:
line 830 also has another beer pun. Just search for the word `beer` on this file and remove both puns, please. Much better for the last one would be `Congratulations!` or anything like that but not `We're done let's have a beer`.

Reply:
Sure - that's fine by me :smile: I'll get that done right away if I can do it on GitHub web, if not tonorrow.